src/dashboards/hacs-repository-dashboard.ts

```
@@ -284,15 +284,15 @@ export class HacsRepositoryDashboard extends LitElement {
           entry.divider
             ? html`<li divider role="separator"></li>`
             : html`
-                <ha-menu-item
+                <ha-md-menu-item
                   class="${entry.error ? "error" : entry.warning ? "warning" : ""}"
                   .clickAction=${() => {
                     entry?.action && entry.action();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)